src/AutorestSwift/View Models/OperationViewModel.swift

```diff
@@ -90,8 +90,7 @@ struct OperationViewModel {
     let returnType: ReturnTypeViewModel?
     let params: OperationParameters
     let pipelineContext: [KeyValueViewModel]?
-    private let requests: [RequestViewModel]?
```

Comment:
What happened to this?

Reply:
we don't need to keep a property of requests array in ObjectionViewmodel since we only consider the first request anywayre.